lib/redis_memo/future.rb

```diff
@@ -28,7 +28,7 @@ def initialize(
   end
 
   def context
-    [@ref, @method_id, @method_args, @depends_on]
+    [@ref, @method_id, @method_args, @dependent_memos]
```

Comment:
Nice! Can we remove the context method and read the attribute `dependent_memos` directly?

Reply:
We still need the method_id while computing the `method_cache_keys`, but context can just be method_id and dependent_memos now